### ISSUE TITLE
Missing the types of Cheerio, CheerioSelector and CheerioStatic

### ISSUE THREAD

**Issue by saltyshiomix** [IonicaBizau/scrape-it#136]
I found no declarations in the `index.d.ts` (`v5.1.4`):

(`Cheerio`, `CheerioSelector` and `CheerioStatic`)

```ts
declare namespace scrapeIt {
    export interface ScrapeOptions {
        [key: string]: string | ScrapeOptionList | ScrapeOptionElement;
    }

    export interface ScrapeOptionElement {
        selector?: string;
        convert?: (value: any) => any;
        how?: string | ((element: CheerioSelector) => any);
        attr?: string;
        trim?: boolean;
        closest?: string;
        eq?: number;
        texteq?: number;
    }

    export interface ScrapeOptionList {
        listItem: string;
        data?: ScrapeOptions;
        convert?: (value: any) => any;
    }

    export interface ScrapeResult<T> {
        data: T,
        $: Cheerio,
        response: any,
        body: string
    }

    export function scrapeHTML<T>(body: CheerioStatic | string, options: ScrapeOptions): T;
}

declare function scrapeIt<T>(url: string | object, opts: scrapeIt.ScrapeOptions): Promise<scrapeIt.ScrapeResult<T>>;

declare function scrapeIt<T>(url: string | object, opts: scrapeIt.ScrapeOptions, cb: (err: any, res: scrapeIt.ScrapeResult<T>) => void): void;
export = scrapeIt;
```

I think we should import types from `cheerio`.
Any other ideas?

In advance, this awesome tool I'm so glad to use :)
Thank you for your great works.

**Comment by saltyshiomix** [comment]
Currently I'm avoiding this issue by adding `@types/cheerio` in my projects' devDependencies.

**Comment by saltyshiomix** [comment]
I think we should add `@types/cheerio` in the **dependencies** instead of devDependencies :)

**Comment by saltyshiomix** [comment]
I tested both `yarn install` and `npm install`, but both of them have same errors.

**Comment by IonicaBizau** [comment]
Probably fixed via #137
